docs/proposals/backup-restore.md

```diff
@@ -128,7 +130,7 @@ Upon success, the output will be:
 
 ```console
 ## <progress_bar>
-Found 2 tag(s): v1, v2 in registry-a.k8s.io/kube-apiserver
+Found 2 tag(s) in registry-a.k8s.io/kube-apiserver: v1, v2
```

Comment:
Just a note: I realized that if `v1` does not exist, the output would look like this:

```console
Found 2 tag(s) in registry-a.k8s.io/kube-apiserver: v1, v2
Error: registry-a.k8s.io/kube-apiserver: v1: not found
```

It might be a confusing when the output says "Found tags ..." and then the error says "not found"

Reply:
You are right, I refactored the implementation. Now it works like this:

```console
oras backup mcr.microsoft.com/oss/v2/oras-project/oras:v1.2.3,not-exist` -o backup.tar
Error: failed to resolve tag "test": mcr.microsoft.com/oss/v2/oras-project/oras:not-exist: not found
```